Source/Particles/MultiParticleContainer.H

```diff
@@ -127,6 +127,29 @@ public:
     */
     std::unique_ptr<amrex::MultiFab> GetZeroChargeDensity(const int lev);
 
+    /**
+     * \brief Deposit charge density.
+     *
+     * \param[in,out] rho vector of charge densities (one pointer to MultiFab per mesh refinement level)
+     * \param[in] relative_t relative time of deposition
+     * \param[in] icomp component of the MultiFab where rho is deposited (old, new)
+     */
+    void
+    DepositCharge (amrex::Vector<std::unique_ptr<amrex::MultiFab> >& rho,
+                   const amrex::Real relative_t, const int icomp = 0);
+
+    /**
+     * \brief Deposit current density.
+     *
+     * \param[in,out] J vector of current densities (one three-dimensional array of pointers
+     *                to MultiFabs per mesh refinement level)
+     * \param[in] dt time step
+     * \param[in] relative_t relative time of deposition
```

Comment:
Discussed offline: here and above in the signature we could make more clear what `_t` is relative to in `relative_t`.
`dt`: let's check which time step (coarse or subcycle) and why (only needed for Esirkepov?)

Reply:
Done by @RemiLehe in https://github.com/ECP-WarpX/WarpX/pull/1828/commits/a573efbc8d56331eba946d799a8b6648c5bbf8e2.